src/SFML.System/Time.cs

```diff
@@ -46,26 +46,49 @@ public struct Time : IEquatable<Time>
         ////////////////////////////////////////////////////////////
         public static Time FromMicroseconds(long microseconds) => sfMicroseconds(microseconds);
 
+        ////////////////////////////////////////////////////////////
+        /// <summary>
+        /// Construct a Time value from a TimeSpan
+        /// </summary>
+        /// <param name="timeSpan">A TimeSpan representing the amount of time to represent</param>
+        /// <returns>Time value constructed from an existing TimeSpan</returns>
+        ////////////////////////////////////////////////////////////
+        public static Time FromTimeSpan(TimeSpan timeSpan) => sfMicroseconds(timeSpan.Ticks / 10);
+
         ////////////////////////////////////////////////////////////
         /// <summary>
         /// Returns the time value as a number of seconds
         /// </summary>
         ////////////////////////////////////////////////////////////
-        public float AsSeconds() => sfTime_asSeconds(this);
+        public float AsSeconds() => microseconds / 1000000f;
 
         ////////////////////////////////////////////////////////////
         /// <summary>
         /// Returns the time value as a number of milliseconds
         /// </summary>
         ////////////////////////////////////////////////////////////
-        public int AsMilliseconds() => sfTime_asMilliseconds(this);
```

Comment:
If we don't use sfTime_asMilliseconds and sfTime_asSeconds, should we still define the CSFML DLL reference at the bottom of the file?

Reply:
It's not hurting anything for them to be there, and they do exist in the CSFML lib, which is why I left them, even unused. I can remove them for clarity sake if we think that's best.